catalyst_voices/lib/pages/account/account_page.dart

```diff
@@ -0,0 +1,225 @@
+        child: Padding(
+          padding: const EdgeInsets.all(32),
+          child: Column(
+            children: [
+              _Header(),
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const SizedBox(height: 32),
+                    _Tab(),
+                    const SizedBox(height: 48),
+                    _KeychainCard(
+                      connectedWallet: 'Lace',
+                      roles: const [
+                        'Voter (Default)',
+                        'Proposer',
+                        'Drep',
+                      ],
+                      onRemoveKeychain: () => debugPrint('Keychain removed'),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _Header extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 350,
+      child: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: CatalystImage.asset(
+                  VoicesAssets.images.accountBg.path,
+                ).image,
+                fit: BoxFit.cover,
+              ),
+            ),
+          ),
+          Positioned(
+            top: 24,
+            left: 8,
+            child: VoicesIconButton.filled(
+              onTap: () {
+                GoRouter.of(context).pop();
+              },
+              style: ButtonStyle(
+                backgroundColor: WidgetStateProperty.all(
+                  Theme.of(context).colors.elevationsOnSurfaceNeutralLv1White,
+                ),
+                foregroundColor: WidgetStateProperty.all(
+                  Theme.of(context).colors.iconsForeground,
+                ),
+              ),
+              child: VoicesAssets.icons.arrowNarrowLeft.buildIcon(),
+            ),
+          ),
+          Positioned(
+            bottom: 48,
+            left: 32,
+            child: Wrap(
+              direction: Axis.vertical,
+              children: [
+                Text(
+                  context.l10n.myAccountProfileKeychain,
+                  style: Theme.of(context).textTheme.displayMedium?.copyWith(
+                        color: Colors.white,
+                      ),
+                ),
+                const SizedBox(height: 4),
+                Text(
+                  context.l10n.yourCatalystKeychainAndRoleRegistration,
+                  style: Theme.of(context).textTheme.titleMedium?.copyWith(
+                        color: Colors.white,
+                      ),
+                ),
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Tab extends StatelessWidget {
```

Comment:
Pls add a const constructor to all these private widgets, then you'll be able to create const instances of them :)